pkg/repo/chartrepo.go

```diff
@@ -185,7 +188,7 @@ func (r *ChartRepository) generateIndex() error {
 
 // FindChartInRepoURL finds chart in chart repository pointed by repoURL
 // without adding repo to repositories
-func FindChartInRepoURL(repoURL, chartName, chartVersion, certFile, keyFile, caFile string, getters getter.Providers) (string, error) {
+func FindChartInRepoURL(repoURL, username, password, chartName, chartVersion, certFile, keyFile, caFile string, getters getter.Providers) (string, error) {
```

Comment:
This is a breaking change since it is a change to a public function signature. We'll probably want to create a new function here that can handle taking in a username and password.

Reply:
Fixed.